### ISSUE TITLE
Resizing app window moves toolbar

### ISSUE THREAD

**Issue by chadrolfs** [adobe-photoshop/spaces-design#1915]
https://www.dropbox.com/s/dg959nfib7ydxjd/toolbar-wander-onresize.mov?dl=0

**Comment by iwehrman** [comment]
This is an old, known core bug. There may be a Watson issue for it. CC @jfitz-adobe

**Comment by iwehrman** [comment]
But, if there's not a Watson issue for it, we should file one.

**Comment by chadrolfs** [comment]
https://watsonexp.corp.adobe.com//#bug=4037335

**Comment by tiburbage** [comment]
This was part of our recent CEF related optimization work.
I've verified in staging builds very close to our integrate-to-main point, and with spaces-design master at abea5cb, that we have accomplished significant improvements in this area. The improvement is most noticeable when no docs are open, and on 1x vs 2x. When documents are open, we get caught up by Photoshop's own event handling which slows us down, and of course on 2x there is just more data to deal with.
